rclcpp/src/rclcpp/subscription.cpp

```diff
@@ -18,6 +18,8 @@
 #include <memory>
 #include <string>
 
+#include <rcutils/logging_macros.h>
+
```

Comment:
nitpick: this should be included after the `rclcpp` headers or the `rmw/*` headers included below should be moved up, again following the Google Style guides recommendation but with some flexibility. In my opinion the only thing that's important is that there is separation between "this project's headers" and "other project's headers" of which the `rclcpp/*` includes and `rmw/*`/`rcutils/*` includes fit into respectively.